webui/react/src/pages/TrialDetails/Profiles/Charts/ThroughputMetricChart.tsx

```diff
@@ -24,7 +25,12 @@ const ThroughputMetricChart: React.FC<ChartProps> = ({ getOptionsForMetrics, tri
 
   return (
     <Section bodyBorder bodyNoPadding title="Throughput">
-      <UPlotChart data={throughputMetrics.data} options={options} />
+      <LineChart
+        series={throughputMetrics.data}
+        xAxis={XAxisDomain.Time}
```

Comment:
In order to get the right timestamps format for the x-axis tick labels, looks like we would need to be able to pass additional info here besides just the name. either that, or make is so where linechart changes the timestamps to this format when the x-range is small.

Reply:
+1 to adapting when x-range is small

Reply:
Talked to @mapmeld, separate it out to a new ticket